base/stats.go

```diff
@@ -1878,6 +1876,10 @@ func (d *DbStats) initDatabaseStats() error {
 	if err != nil {
 		return err
 	}
+	resUtil.TombstoneCount, err = NewIntStat(SubsystemDatabaseKey, "tombstone_count", StatUnitNoUnits, TombstoneCount, StatAddedVersion4dot0dot0, StatDeprecatedVersionNotDeprecated, StatStabilityInternal, labelKeys, labelVals, prometheus.CounterValue, 0)
```

Comment:
`Desc` description const suffix

```suggestion
	resUtil.TombstoneCount, err = NewIntStat(SubsystemDatabaseKey, "tombstone_count", StatUnitNoUnits, TombstoneCountDesc, StatAddedVersion4dot0dot0, StatDeprecatedVersionNotDeprecated, StatStabilityInternal, labelKeys, labelVals, prometheus.CounterValue, 0)
```